src/thx/State.hx

```diff
@@ -0,0 +1,41 @@
+package thx;
```

Comment:
Cool, I will merge once it is moved.

On Wed, Nov 18, 2015 at 11:22 AM Kris Nuttycombe <notifications@github.com>
wrote:

> In src/thx/State.hx
> <https://github.com/fponticelli/thx.core/pull/111#discussion_r45238065>:
>
> > @@ -0,0 +1,41 @@
> > +package thx;
>
> It can be in thx.fp, that's fine.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fponticelli/thx.core/pull/111/files#r45238065>.
>